t/t2703-mini-bulksubmit.t

```diff
@@ -253,8 +253,8 @@ test_expect_success 'flux-mini bulksubmit reports invalid replacement strings' '
 	test_expect_code 1 flux mini bulksubmit -n {1} {} ::: 0 1 2 \
```

Comment:
commit message in 1761edaae25bf57dc46a22a55779adc06796441e

s/indeces/indices/